forgettable-frontend/src/pages/CompanyListPage/CompanyListPage.js

```diff
@@ -0,0 +1,180 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
is there any React import usage in this file? Import looks redundant

Reply:
I think we need the imports since they are not a functional component - gave me an error when i tried running without it